package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "vscode-mezzurite",
+  "name": "mezzurite",
```

Comment:
We should leave this as `vscode-mezzurite`. This isn't the package that contains `mezzurite`.

Reply:
Yeah! But, while publishing vscode-extension, it compares the name if already present then updates version and extension. Otherwise, creates a new extension, which is not what we want

Reply:
That sounds like a problem with the configuration we have in the marketplace. We should fix it in both places.